jscomp/core/bs_conditional_initial.ml

```diff
@@ -64,16 +66,22 @@ let setup_env () =
     Matching_polyfill.names_from_construct_pattern;
 #ifndef BS_RELEASE_BUILD
     Printexc.record_backtrace true;
-    (* (let root_dir = *)
-        (* Filename.dirname *)
-          (* (Filename.dirname Sys.executable_name) in *)
-    (* let (//) = Filename.concat in *)
-    (* Clflags.include_dirs := *)
-      (* (root_dir//"jscomp"//"others") :: *)
-      (* (root_dir//"jscomp"//"stdlib-412") :: *)
-      (* (root_dir//"jscomp"//"runtime") :: *)
-      (* !Clflags.include_dirs); *)
+    (let jscomp =
+        (* jscomp/main/bsc.exe -> jscomp *)
+        Filename.dirname
+          (Filename.dirname Sys.executable_name) in
+    Clflags.include_dirs :=
+      (jscomp//"others") ::
+      (jscomp//"stdlib-412/stdlib_modules") ::
+      (jscomp//"stdlib-412") ::
+      (jscomp//"runtime") ::
+      !Clflags.include_dirs);
+#endif
+
+#ifdef BS_RELEASE_BUILD
```

Comment:
Source to ensure we define this

https://github.com/melange-re/melange/blob/fork/dune

Reply:
This is just notes to others who see this PR, like myself in 5 min